src/main/scala/io/iohk/ethereum/domain/appstate/BestBlockInfo.scala

```diff
@@ -0,0 +1,5 @@
+package io.iohk.ethereum.domain.appstate
+
+import akka.util.ByteString
+
+case class BestBlockInfo(hash: ByteString, number: BigInt)
```

Comment:
Maybe this could be called just `BlockInfo`? Just to make it more generic. I think I need to use this in ETCM-1081